src/DotNetWorker.Grpc/Definition/GrpcFunctionDefinition.cs

```diff
@@ -51,5 +54,23 @@ public GrpcFunctionDefinition(FunctionLoadRequest loadRequest, IMethodInfoLocato
         public override IImmutableDictionary<string, BindingMetadata> OutputBindings { get; }
 
         public override ImmutableArray<FunctionParameter> Parameters { get; }
+
+        private Dictionary<string, object> GetAdditionalPropertiesDictionary(ParameterInfo parameterInfo)
+        {
+            Dictionary<string, object> properties = new();
+
+            // Get the input converter attribute information, if present on the parameter.
+            var inputConverterAttribute = parameterInfo?.GetCustomAttribute<InputConverterAttribute>();
+
+            if (inputConverterAttribute != null)
+            {
+                properties = new Dictionary<string, object>()
+                {                     
+                    { PropertyBagKeys.ConverterType, inputConverterAttribute.ConverterType.AssemblyQualifiedName }
```

Comment:
Can't pass the `System.Type` (inputConverterAttribute.ConverterType) here in the properties dictionary. This is because we serialize the function definition in the [OnFunctionLoaded method](https://github.com/Azure/azure-functions-dotnet-worker/blob/main/src/DotNetWorker.Grpc/Http/GrpcWorkerDiagnostics.cs#L68). An object with a property of type `System.Type` cannot be serialized because accessing the `DeclaringMethod` prop of a System.Type instance just throws.

The below code will throw.
```
var myInstance = new ClasWithTypeProperty();
myInstance.TheData = typeof(Customer);

var str = System.Text.Json.JsonSerializer.Serialize(myInstance);
```


This is why I have to recreate the type using `Type.GetType` from the fully qualified name inside the DefaultInputConversionFeature.